Tensile/Tests/nightly/convolution_config/test_simple.py

```diff
@@ -0,0 +1,82 @@
+import pytest
```

Comment:
Great!

Other unit tests are so far in the Tensile/UnitTests directory.  They run both with nightly and with pre-checkin since they run very quickly (Reported time is under 1s).

Reply:
i'll move em over there soon